starter-code/server.js

```diff
@@ -19,7 +19,7 @@ app.use(express.static('./public'));
 
 
 // COMMENT: What is this function doing? Why do we need it? Where does it receive a request from?
-// (put your response in a comment here)
+//This is a proxy of the request function. We need it beacause disguise the route we are useing to use the github api.We are receiving a request from the app.get route for github right below this.
```

Comment:
I wouldn't say the purpose is to disguise how we are using the GitHub API (after all, requests are directly passed through). We just need to add the `Authorization` header with our secret token.